src/servlets/utils/helpers.ts

```diff
@@ -47,6 +57,65 @@ export const getImageUrl = (cid: string, gateway: string | null): string => {
   return `${gateway}${cid}`
 }
 
+type ExploreInfoType = {
+  title: string,
+  description: string,
+  image: string
+}
+export const getExploreInfo = (type: string): ExploreInfoType  => {
+  const imageMap: { [key: string]: ExploreInfoType } = {
```

Comment:
In practice probably nbd but I would define this map as a const outside the function so it's initialized once and only once